src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -15,6 +15,10 @@
             <GroupBox x:Uid="AnalysisGroupBox"
                       Header="{x:Static local:AdvancedOptionPageStrings.Option_Analysis}">
                 <StackPanel>
+                    <CheckBox x:Name="Enable_power_save_mode"
+                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Enable_power_save_mode}" />
+                    <CheckBox x:Name="Disable_analyzers"
+                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Disable_analyzers}" />
```

Comment:
if the user doesn't want analyzers running, i think we should be true to that.  that means that IDE features that are analyzers (including remove unused usings) shouldn't run.  You just get compiler errors.